mypyc/doc/native_classes.rst

```diff
@@ -38,7 +38,7 @@ can be assigned to (similar to using ``__slots__``)::
         def method(self) -> None:
             self.z = "x"
 
-    o = Cls()
+    o = Cls(0)
```

Comment:
This allowed this piece of code to compile for me (`__init__` is expecting one non-self argument).